RpcSystemAssetTracker/RpcSystemAssetTrackerPlugin.cs

```diff
@@ -36,6 +42,7 @@ public override void Configure()
                 var dbPath = GetConfiguration().GetSection("DBPath").Value ?? "SystemAssetBalanceData";
                 _db = DB.Open(Path.GetFullPath(dbPath), new Options { CreateIfMissing = true });
                 _shouldTrackUnclaimed = (GetConfiguration().GetSection("TrackUnclaimed").Value ?? true.ToString()) != false.ToString();
+                _shouldTrackHistory = (GetConfiguration().GetSection("TrackHistory").Value ?? true.ToString()) != false.ToString();
```

Comment:
This works?, it use lowercase and by default, false.ToString() returns `False` I prefer to avoid the comparison of booleans as strings